packages/wrapped-keys/src/lib/lit-actions-client/constants.ts

```diff
@@ -0,0 +1,18 @@
+import { LitCidRepository } from './types';
+
+const LIT_ACTION_CID_REPOSITORY: LitCidRepository = {
+  signTransaction: Object.freeze({
+    evm: 'QmdYUhPCCK5hpDWMK1NiDLNLG6RZQy61QE4J7dBm1Y2nbA',
+    solana: 'QmSi9GL2weCFEP1SMAUw5PDpZRr436Zt3tLUNrSECPA5dT',
+  }),
+  signMessage: Object.freeze({
+    evm: 'QmTMGcyp77NeppGaqF2DmE1F8GXTSxQYzXCrbE7hNudUWx',
+    solana: 'QmUxnWS8VU9QwZRdyfwsEtvhJZcvcdrannjokgZoA1sesy',
+  }),
+  generateEncryptedKey: Object.freeze({
+    evm: 'QmaoPMSqcze3NW3KSA75ecWSkcmWT1J7kVr8LyJPCKRvHd',
+    solana: 'QmdRBXYLYvcNHrChmsZ2jFDY8dA99CcSdqHo3p1ES3UThL',
+  }),
+};
+
```

Comment:
if we are freezing the all properties on `LIT_ACTION_CID_REPOSITORY` should we also freeze the top level object such that it cannot be modified at runtime? const will be dropped at runtime and allow for modifications on the root object so we might want to consider a top level freeze as well.